apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/CustomItemAdapter.java

```diff
@@ -0,0 +1,63 @@
+package com.microsoft.appcenter.sasquatch.activities;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageButton;
+import android.widget.ListAdapter;
+import android.widget.TextView;
+
+import com.microsoft.appcenter.sasquatch.R;
+
+import java.util.ArrayList;
+
+public class CustomItemAdapter extends BaseAdapter implements ListAdapter {
+    private ArrayList<String> list = new ArrayList<String>();
+    private Context context;
+
+    public CustomItemAdapter(ArrayList<String> list, Context context){
+        this.list = list;
```

Comment:
remove this after renaming to `m{Field}`

Reply:
Fix in new commit.